grr/server/grr_response_server/gui/ui/lib/api/http_api_service.ts

```diff
@@ -223,6 +223,14 @@ export class HttpApiService {
         url, {client_ids: [clientId], labels: [label]});
   }
 
+  removeClientLabel(clientId: string, label: string): Observable<HttpResponse<{}>> {
+    const url = `${URL_PREFIX}/clients/labels/remove`;
+    return this.http.post<{}>(
+        url, {client_ids: [clientId], labels: [label]}, {
+          observe: 'response',
```

Comment:
Im on the run right now, cannot look it up: What does observe response do?

Reply:
By default, the returned observable would be the data we are requesting for. In these case, I wanted to get the status, and in order to do that I had to specify `response` to be returned.